README.md

```diff
@@ -39,19 +39,14 @@ With the ESnext field declarations proposal, the above example can be written as
 class Counter extends HTMLElement {
```

Comment:
I just want to confirm that these changes are functionally equivalent.

**Before**:

<img width="484" alt="screen shot 2017-05-25 at 3 13 49 pm" src="https://cloud.githubusercontent.com/assets/1500684/26466471/d9a8b4b0-415c-11e7-835d-10892dd34085.png">

**After**:

<img width="450" alt="screen shot 2017-05-25 at 3 13 34 pm" src="https://cloud.githubusercontent.com/assets/1500684/26466478/de261a5a-415c-11e7-9a1e-6f17ceeca596.png">


This is a _very_ common pattern in React currently and based on what I see, this appears to still be supported. Just want to clarify that :) Is that correct?